draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -851,6 +853,20 @@ To prevent these types of linkability, various methods, including but not limite
 - Use advanced cryptographic schemes, outside the scope of this specification.
 - Issue a batch of SD-JWTs to the Holder to enable the Holder to use a unique SD-JWT per Verifier. This only helps with Verifier/Verifier unlinkability.
 
+Colluding Verifier/Verifier pairs correspond to RP+RP'-U Unlinkability as defined in [@ISO.27551].
+
+## Issuer Identifier
+
+Issuer issuing only one type of SD-JWT might have privacy implications, because if the Holder has an SD-JWT issued by that Issuer, its type and claim names can be determined.
+
+For example, if the National Cancer Institute only issued SD-JWTs with cancer registry information, it is possible to deduce that the Holder owning its SD-JWT is a cancer patient.
+
+Moreover, issuer identifier alone may reveal information about the user. 
+
+For example, when a military organization or a drug rehab center issues a vaccine credential, verifiers can deduce that the holder is a military member or has a substance use disorder.
```

Comment:
updated the text because confirmed with the smart health community for below use-case:
> A scenario we have seen is something like a military organization or a rehab clinic issuing a vaccine credential; if the iss is directly associated with one of theses orgs, it allow verifiers to deduce that the holder is a military member or has a substance use disorder. That's undesirable, and indeed we've seen issuers adopt generic names or maybe even (I'm not sure about this) join up with other issuing orgs to pool populations.

Reply:
makes sense